packages/astroport-governance/src/assembly.rs

```diff
@@ -139,6 +153,32 @@ impl Config {
             )));
         }
 
+        if self.proposal_effective_delay < MINIMUM_DELAY {
+            return Err(StdError::generic_err(format!(
+                "The effective delay for a proposal cannot be less than {} blocks.",
+                MINIMUM_DELAY
+            )));
+        }
+
+        if self.proposal_expiration_period < MINIMUM_EXPIRATION_PERIOD {
+            return Err(StdError::generic_err(format!(
+                "The expiration period for a proposal cannot be less than {} blocks.",
+                MINIMUM_EXPIRATION_PERIOD
+            )));
+        }
+
+        for whitelisted_link in &self.whitelisted_links {
+            if !whitelisted_link
+                .chars()
+                .all(|c| char::is_ascii(&c) && !['<', '>', ' ', '\\', '{', '}'].contains(&c))
```

Comment:
Create constant array before 170 line.

Reply:
I suggest to validate a link before it is added to Config. In the current implementation it will always iterate and validate other validated links.

Reply:
Thanks, it makes sense. Done.